src/EntropyGenerator/Uniq.php

```diff
@@ -4,7 +4,7 @@
 class Uniq implements EntropyGeneratorInterface
 {
     /**
-     * {@inheritdoc}
+     * @return string
```

Comment:
Thats something, ZendStudio adds when creating a phpdoc from a parent-method. Here is a discussion at [PHPStorm Bug-Tracker](https://youtrack.jetbrains.com/issue/WI-23395#id_l.I.ic.it.c.c_27_740760.comment_content_row_27_740760).

Ignoring that, @inheritDoc is not useful for simpler IDEs oder Texteditors, since the reason why a doc-block was created is to make a statement about what type each parameter will accept. And to be hornest: [PHPStorm add that all automatically](https://drupalize.me/videos/generating-code-phpstorm?p=2017) (If a variable's type has been declared somewhere, PHPStorm will figure that out and form proper doc-blocks).